kratos/python/add_model_part_to_python.cpp

```diff
@@ -791,6 +796,7 @@ void AddModelPartToPython(pybind11::module& m)
         .def("NumberOfElements", &ModelPart::NumberOfElements)
         .def("NumberOfConditions", ModelPartNumberOfConditions1)
         .def("NumberOfConditions", &ModelPart::NumberOfConditions)
+        .def("NumberOfGeometries", ModelPartNumberOfGeometries1)
```

Comment:
```suggestion
        .def("NumberOfGeometries", &ModelPart::NumberOfGeometries)
```
Why this doesn't work?